packages/realm-web/types/realm/app.d.ts

```diff
@@ -360,6 +360,11 @@ declare namespace Realm {
      * Specifying this will enable the server to respond differently to specific versions of specific apps.
      */
     app?: LocalAppConfiguration;
+
+    /**
+     * Specify where synced Realms and metadata is stored. If not specified, the current work directory is used.
+     */
+    baseFilePath?: string;
```

Comment:
No need to add this, as `realm-web` don't have an implementation for it and it's the only component depending on these types now.

Reply:
True, it can be removed.